jsapp/js/account/accountFieldsEditor.component.tsx

```diff
@@ -0,0 +1,281 @@
+            error={props.errors?.gender}
+          />
+        </div>
+      )}
+
+      {/* Bio */}
+      {isFieldToBeDisplayed('bio') && (
+        <div className={styles.row}>
+          <TextBox
+            label={getLabel('bio')}
+            value={props.values.bio}
+            onChange={onAnyFieldChange.bind(onAnyFieldChange, 'bio')}
+            errors={props.errors?.bio}
+          />
+        </div>
+      )}
+
+      {/* Country */}
+      {isFieldToBeDisplayed('country') && (
+        <div className={styles.row}>
+          <label htmlFor='country'>{getLabel('country')}</label>
+          <KoboSelect
+            name='country'
+            type='outline'
+            size='l'
+            isClearable
+            isSearchable
+            selectedOption={props.values.country}
+            onChange={(value: string | null) =>
+              onAnyFieldChange('country', value || '')
+            }
+            options={envStore.data.country_choices}
+            error={props.errors?.country}
+          />
+        </div>
+      )}
+
+      {/* City */}
+      {isFieldToBeDisplayed('city') && (
+        <div className={styles.row}>
+          <TextBox
+            label={getLabel('city')}
+            value={props.values.city}
+            onChange={onAnyFieldChange.bind(onAnyFieldChange, 'city')}
+            errors={props.errors?.city}
+          />
+        </div>
+      )}
+
+      {/* Social */}
+      {(isFieldToBeDisplayed('twitter') ||
+        isFieldToBeDisplayed('linkedin') ||
+        isFieldToBeDisplayed('instagram')) && (
+        <>
+          <label>{t('Social')}</label>
+
+          {/* Twitter */}
+          {isFieldToBeDisplayed('twitter') && (
+            <div className={styles.row}>
+              <TextBox
+                startIcon='logo-twitter'
+                placeholder={getLabel('twitter')}
+                value={props.values.twitter}
+                onChange={onAnyFieldChange.bind(onAnyFieldChange, 'twitter')}
+                errors={props.errors?.twitter}
+              />
+            </div>
+          )}
+
+          {/* LinkedIn */}
+          {isFieldToBeDisplayed('linkedin') && (
+            <div className={styles.row}>
+              <TextBox
+                startIcon='logo-linkedin'
+                placeholder={getLabel('linkedin')}
+                value={props.values.linkedin}
+                onChange={onAnyFieldChange.bind(onAnyFieldChange, 'linkedin')}
+                errors={props.errors?.linkedin}
+              />
+            </div>
+          )}
+
+          {/* Instagram */}
+          {isFieldToBeDisplayed('instagram') && (
+            <div className={styles.row}>
+              <TextBox
+                startIcon='logo-instagram'
+                placeholder={getLabel('instagram')}
+                value={props.values.instagram}
+                onChange={onAnyFieldChange.bind(onAnyFieldChange, 'instagram')}
+                errors={props.errors?.instagram}
```

Comment:
The `errors` prop of all the `TextBox` components has a type mismatch - passing `undefined` when the prop is expecting `false`. One option:

```code
errors={props.errors?.instagram || false}
```

Reply:
Do you see this being raised as type mismatch in command line or in editor? I don't have any errors when running webpack. My Sublime Text is also handling this without problems:
<img width="652" alt="Screenshot 2023-11-02 at 17 12 42" src="https://github.com/kobotoolbox/kpi/assets/2521888/cd18c808-d75e-40fb-8389-bd724113f548">